images/build/cross/Makefile

```diff
@@ -18,8 +18,8 @@ STAGING_REGISTRY?=gcr.io/k8s-staging-build-image
 PROD_REGISTRY?=us.gcr.io/k8s-artifacts-prod/build-image
 IMAGE=kube-cross
 
-TAG?=kubernetes-$(shell git describe --tags --match='v*' --abbrev=14)
-KUBE_CROSS_VERSION=$(shell cat VERSION)
+TAG?=$(shell git describe --tags --always --dirty)
```

Comment:
we should probably leave `--abbrev=14` in

Reply:
otherwise the sha suffix will not be of the same length everytime

Reply:
@dims -- I did that to more closely match the tag that the image builder spits out: https://github.com/kubernetes/test-infra/blob/4fd8fb189fc644bc995da1f73a0991751a367745/images/builder/main.go#L49

Reply:
Ultimately, the tag we care about here is `VERSION`; `TAG` is more of a sanity check so we have an idea of what we're promoting compared to `VERSION`.

Reply:
we'll know soon enough if something breaks i guess